R/DBConnection.R

```diff
@@ -274,11 +281,16 @@ setMethod("dbGetQuery", signature("DBIConnection", "character"),
 #' Execute an update statement, query number of rows affected, and then close result set
 #'
 #' Executes a statement and returns the number of rows affected.
-#' `dbExecute()` comes with a default implementation
+#' [dbExecute()] comes with a default implementation
```

Comment:
I try to add a link link only the first occurrence of a function on a help page.